servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/SslCertificateCompressionTest.java

```diff
@@ -56,6 +58,8 @@ class SslCertificateCompressionTest {
     @ParameterizedTest(name = "{displayName} [{index}] {arguments}")
     @ValueSource(booleans = {true, false})
     void negotiatesCertificateCompression(boolean clientEnabled) throws Exception {
+        assumeTrue(OpenSsl.isAvailable());
```

Comment:
FYI. You can also use `@EnabledIf`

Reply:
Good idea! At the moment (if my grep foo works) it looks like `assumeTrue` is exclusively used in ServiceTalk, I'll check if there is a benefit migrating to the annotation.

Reply:
I suspect this is just because they did migrate from junit4 and did not update these yet.

Reply:
1. Should we use `SslProvider.isOptionSupported(OPENSSL, OpenSslContextOption.CERTIFICATE_COMPRESSION_ALGORITHMS)` instead of `OpenSsl.isAvailable()`?
2. Consider adding a message that will clarify why the test is disabled.